DOCSP-35923: csot page (#657)
mongodb/docs-java: source/crud/read-operations/cursor.txt

Overview
--------

> In this guide, you can learn how to access data using a **cursor** in

Reviewer: ```suggestion
In this guide, you can learn how to access data by using a **cursor** in
```

matched by your search criteria and to further specify which documents
to see by setting parameters through methods.

.. tip:: Cursor Timeout

>    You can set a timeout to limit the amount of time it takes to
>    retrieve data from a cursor. To learn more, see the

Reviewer: S: this sort of sounds like a free way to speed up operations

```suggestion
   You can set a timeout for retrieving documents from a cursor.
   To learn more, see the
```

applies to similar language in other tips
